install/jsf/docs/index.html

```diff
@@ -1,70 +1,113 @@
-<span class="emphasize">Your Starting Point</span></h1>
+    <h1>Welcome to the Jakarta Server Faces Test Compatibility Kit, Version 3.0.0<br>
+        <span class="emphasize">Your Starting Point</span></h1>
 </div>
 <hr>
-<h2> JavaServer Faces TCK 2.3 Documentation</h2>
-<p>The  JavaServer Faces TCK 2.3 documentation includes the following:      </p>
+<h2> Jakarta Server Faces TCK 3.0.0 Documentation</h2>
+<p>The Jakarta Server Faces TCK 3.0.0 documentation includes the following: </p>
 <ul>
 
-  <li><p>The <a href="ReleaseNotes.html"> JavaServer Faces
-  Release Notes</a> provide late-breaking information about the
-  JavaServer Faces TCK, Version 2.3.</p></li> <li><p>The <em> JavaServer
-  Faces TCK User's Guide </em> provides the information that you need to
-  install, set up, and run the JavaServer Faces TCK, Version 2.3. In
-  addition, the guide also provides the rules with which you must comply
-  to pass the JavaServer Faces TCK.</p></li>
-
-  <li><p>The <a href="assertions/JSFJavadocAssertions.html">Javadoc Assertion
-  List</a> lists all the javadoc assertions that are indirectly tested by
-  the JavaServer Faces TCK.</p></li>
-  <li><p>The <a href="assertions/JSFSpecAssertions.html">Specification
-  Assertion List</a> lists all the specification assertions that are tested
-  by the JavaServer Faces TCK.</p></li>
+    <li><p>The <a href="ReleaseNotes.html"> Jakarta Server Faces
+        Release Notes</a> provide late-breaking information about the
+        Jakarta Server Faces TCK, Version 3.0.0.</p></li>
+    <li><p>The <em> Jakarta Server
+        Faces TCK User's Guide </em> provides the information that you need to
+        install, set up, and run the Jakarta Server Faces TCK, Version 3.0.0. In
+        addition, the guide also provides the rules with which you must comply
+        to pass the Jakarta Server Faces TCK.</p></li>
+
+    <li><p>The <a href="assertions/JakartaServerFacesJavadocAssertions.html">Javadoc Assertion
+        List</a> lists all the javadoc assertions that are indirectly tested by
+        the Jakarta Server Faces TCK.</p></li>
+    <li><p>The <a href="assertions/JakartaServerFacesSpecAssertions.html">Specification
+        Assertion List</a> lists all the specification assertions that are tested
+        by the Jakarta Server Faces TCK.</p></li>
 </ul>
 
 <h2>JT Harness Documentation</h2>
-<p>The online version of the JT Harness version 5.0 documentation is available <a href="https://wiki.openjdk.java.net/display/CodeTools/Documentation">here</a>.</p>
+<p>The online version of the JT Harness version 5.0 documentation is available <a
+        href="https://wiki.openjdk.java.net/display/CodeTools/Documentation">here</a>.</p>
 
 <hr>
-<p><small><address>Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.</address></small></p>
+<p><small>
+    <address>Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.</address>
```

Comment:
Please update second date from 2018 to 2020

Reply:
thanks !